inference-engine/src/transformations/src/transformations/common_optimizations/add_fake_quantize_fusion.cpp

```diff
@@ -21,7 +22,14 @@ ngraph::pass::AddFakeQuantizeFusion::AddFakeQuantizeFusion() {
     auto input_pattern = ngraph::pattern::any_input();
     auto const_pattern = ngraph::pattern::wrap_type<opset5::Constant>();
     auto add_pattern = ngraph::pattern::wrap_type<opset5::Add>({input_pattern, const_pattern},
-                                                               pattern::consumers_count(1));
+            [] (const Output<Node>& node) -> bool {
+                if (node.get_target_inputs().size() != 1)
+                    return false;
+                const auto& inputs = node.get_node()->input_values();
+                const auto& node_type_info = inputs[0].get_node()->get_type_info();
+                return node_type_info != opset5::Convolution::type_info &&
```

Comment:
Suggest checking on MatMul node too

Reply:
Also (on the future, maybe) ConvolutionBackpropData, GroupConvolutionBackpropData